doc/api/modules.md

```diff
@@ -626,9 +626,11 @@ added: v8.9.0
 -->
 
 * `request` {string} The module path whose lookup paths are being retrieved.
-* Returns: {Array}
+* Returns: {Array|null}
 
-Returns an array containing the paths searched during resolution of `request`.
+Returns an array containing the paths searched during resolution of `request` or
+null if the `request` string references a core module, for example `http` or
```

Comment:
Maybe `null` -> `` `null` ``?

Reply:
should the mention of array and string also be highlighted then? I viewed all of those as the same, and since array and string were not highlighted in the previous version, I used that as a guide. Can for sure give them the treatment if it's considered needed.

Reply:
I am also not sure: array and string may have a general meaning, while `null` is more strict. But let us wait what others think. I am OK with the current form as well)